readthedocs/builds/querysets.py

```diff
@@ -149,12 +154,22 @@ def concurrent(self, project):
             # The project has translations, counts their builds as well
             query |= Q(project__in=project.translations.all())
 
+        # If the project belongs to an organization, count all the projects
+        # from this organization as well
+        organization = project.organizations.first()
+        if organization:
+            query |= Q(project__in=organization.projects.all())
+
         concurrent = (
             self.filter(query)
             .exclude(state__in=[BUILD_STATE_TRIGGERED, BUILD_STATE_FINISHED])
-        ).count()
+        ).distinct().count()
 
-        max_concurrent = project.max_concurrent_builds or settings.RTD_MAX_CONCURRENT_BUILDS
+        max_concurrent = (
+            project.max_concurrent_builds or
+            (organization.max_concurrent_builds if organization else 0) or
```

Comment:
Do we want `else 0` here? Will the 0 cause the `or` to move to the default? It seems a bit confusing.

Reply:
Yes, this may be a little tricky. Basically the `0` here works as `False` and makes the `or` to continue to the next value:

```
In [27]: None or 0 or 5
Out[27]: 5
```